src/plot.jl

```diff
@@ -504,3 +504,21 @@ function Base.show(io::IO, p::Plot)
         print_title(io, border_padding, p.xlabel, p_width = border_length)
     end
 end
+
+"""
+    savefig(p, fn[, color])
```

Comment:
We use `;` to indicate keyword argument, and with default values we meant it to be optional so no need to add extra `[]`.

Is `fn` a commonly used name in Plots? For me `filename` is clearer.

```suggestion
    savefig(p, filename; color=false)
```